tests/views/resource_tests.py

```diff
@@ -63,6 +64,18 @@ def setUpTestData(cls):
             edit.userid = user.id
             edit.save()
         cls.resource = Resource.objects.get(pk=cls.resource_instance_id)
+        cls.graph = Graph(cls.resource.graph_id)
+
+    def test_resource_editor_view(self):
```

Comment:
I couldn't figure out a way to get this test to fail on the target branch. It might be that the test graph doesn't have widgets added. But this does seem to exercise the added lines for cards though, which is where we saw the failure.